src/VoteTouchpoints.css

```
@@ -0,0 +1,3 @@
+.vote-touchpoints-margins {
```

```suggestion
.fba-inline-container {
    margin: 3rem 0 0 0 ;
}
```

I think we can just target this class and not have to make a new one